webpack/ForemanTasks/Components/TasksTable/TasksTableConstants.js

```diff
@@ -1,7 +1,7 @@
 import { getControllerSearchProps } from 'foremanReact/constants';
 
 export const TASKS_TABLE_ID = 'TASKS_TABLE';
-export const TASKS_CONTROLLER = 'tasks';
+export const TASKS_CONTROLLER = 'foreman_tasks/tasks';
 export const TASKS_SUCCESS = 'TASKS_SUCCESS';
 export const TASKS_REQUEST = 'TASKS_REQUEST';
 export const TASKS_FAILURE = 'TASKS_FAILURE';
```

Comment:
These are also not being used, as the api uses TASKS_TABLE_ID for the name and createTableActionTypes for the success,request, failure